core/src/main/scala/scalaz/MonadTrans.scala

```diff
@@ -15,6 +15,9 @@ trait MonadTrans[F[_[_], _]] {
   /** The [[scalaz.Monad]] implied by this transformer. */
   implicit def apply[G[_] : Monad]: Monad[F[G, ?]]
 
+  def mapF[G[_], A, B](fa: F[G, A])(f: A => G[B])(implicit M: Monad[G]): F[G, B] =
+    Monad[F[G, ?]].bind(fa)(a => liftM(f(a)))
+
```

Comment:
We have `liftM` and `mapF`, where both `-M` and `-F` refer to type parameter named `G`. This can be confusing :man_shrugging: